### PR TITLE
Support as_ptr on a register for DMA and such

### PR DESCRIPTION
**Pull request by petrohi**

### CODE REVIEW

File: src/generate/generic.rs [hunk 1]
```diff
@@ -32,6 +32,21 @@ pub struct Reg<U, REG> {
 
 unsafe impl<U: Send, REG> Send for Reg<U, REG> {}
 
+impl<U, REG> Reg<U, REG>
+where
+    U: Copy,
+{
+    /// Gets a memory address of a register.
```
Comment (Emilgardis) [inline src/generate/generic.rs:39]: Wording, maybe something like

    Returns the underlying memory address of register

Comment (petrohi) [reply src/generate/generic.rs]: Adjusted.

**Comment by burrbull** [comment]
bors try

**Comment by therealprof** [comment]
@burrbull No need to "try" anymore. GHA is automatically running CI on every commit, cf. https://github.com/rust-embedded/svd2rust/runs/653370311

**Comment by burrbull** [comment]
reflex :smile:

**Comment by therealprof** [comment]
bors r=Emilgardis

**Pull request merged**